patches/tModLoader/Terraria/Player.cs.patch

```diff
@@ -3700,3 +3700,24 @@
  				if (buffType[i] >= type && buffType[i] < type + 3)
  					DelBuff(i);
  			}
+@@ -38411,6 +_,20 @@
+ 
+ 			if (list != null)
+ 				TouchedTiles = list;
++		}
++
++		public Item[] GetPlayerChest() {
```

Comment:
This should either be in Player.TML.cs, an extension method, or in an util class.

Reply:
entirely agree, just gonna remove it for now.